adapters/adnuntius/adnuntius.go

```diff
@@ -109,13 +121,41 @@ func makeEndpointUrl(ortbRequest openrtb2.BidRequest, a *adapter) (string, []err
 		q.Set("gdpr", gdpr)
 		q.Set("consentString", consent)
 	}
+
+	if deviceExt.NoCookies {
+		q.Set("noCookies", "true")
+	}
+
 	q.Set("tzo", fmt.Sprint(tzo))
 	q.Set("format", "json")
 
 	url := a.endpoint + "?" + q.Encode()
 	return url, nil
 }
 
+func getImpSizes(imp openrtb2.Imp) [][]int64 {
+	if imp.Banner == nil {
+		return nil
+	}
+
+	if len(imp.Banner.Format) > 0 {
+		sizes := make([][]int64, len(imp.Banner.Format))
+		for i, format := range imp.Banner.Format {
+			sizes[i] = []int64{format.W, format.H}
+		}
+
+		return sizes
+	}
+
+	if imp.Banner.W != nil && imp.Banner.H != nil {
+		size := make([][]int64, 1)
+		size[0] = []int64{*imp.Banner.W, *imp.Banner.H}
+		return size
+	}
```

Comment:
I noticed that your tests don't seem to cover this code block, could you update the tests so that this is covered?

Reply:
Added test for this as well,